packages/twenty-server/src/engine/core-modules/guard-manager/guard-manager.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from '@nestjs/common';
+
+import { GuardErrorManagerService } from 'src/engine/core-modules/guard-manager/services/guard-error-manager.service';
+import { DomainManagerModule } from 'src/engine/core-modules/domain-manager/domain-manager.module';
+
+@Module({
+  imports: [DomainManagerModule],
+  providers: [GuardErrorManagerService],
+  exports: [GuardErrorManagerService],
```

Comment:
Why not introducing this module it's not a "guard-manager". TBH feels a bit overkill to me too. Not strongly against it though!